pkg/ldd/ldd.go

```diff
@@ -61,7 +62,7 @@ func resolve(library string, library_path string) ([]string, error) {
 	return finalFiles, nil
 }
 
-func ldd(library string, library_path string) (discovered []string, err error) {
+func ldd(library string, library_path []string) (discovered []string, err error) {
```

Comment:
would it make sense to move out the libs loader part and pass in the result of `bin.ImportedLibraries()` to the function as well? It should then be very easy to add a few unit tests.

Reply:
@rmohr if you think that'd be a good idea, I can definitely look into it. I don't currently understand bazeldnf well enough to have an opinion of my own O:-)

As for the testing part, I wanted to add coverage but I was unable to do that quickly/effectively so I sort of gave up. I have reported my thoughts on the status of the test suite in https://github.com/rmohr/bazeldnf/issues/23